### PR TITLE
clean dbclient

### PR DESCRIPTION
**Pull request by mdonaka**
# Summary
clean dbclient

# Purpose
refactoring

# Contents
- Move `value_table` and `join_table` to `job/`
- Add error message
- Add filesystem path

# Testing Methods Performed
- CI

### CODE REVIEW

File: packages/server/computation_container/client/computation_to_db/client.cpp [hunk 1]
```diff
@@ -44,60 +42,50 @@ std::vector<nlohmann::json> convertSchemaVectorToJsonVector(
...
+    Client::getInstance()->writeResultDB(job_uuid, data, data_type, piece_id);
 
     ++piece_id;
     current_size = 0;
     piece_data.clear();
+
+    if (fin)
+    {
+        Client::getInstance()->updateJobCompleted(job_uuid);
+    }
```
Comment (mdonaka) [inline packages/server/computation_container/client/computation_to_db/client.cpp:75]: `auto completed(){}`

**Review by mdonaka — COMMENTED** [review]
memo
`job/table_data/` to `value_table/`

**Comment by mdonaka** [comment]
@watason 
Thank you for your review!
Colud you check the corrected version.

**Review by watason — APPROVED** [review]
PR is OK.
just occuread to me.
i think dbclient should not to be a singleton.
it is not advantage to making singlton.
eather is fine,though.

**Pull request merged**